plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -872,6 +872,9 @@ else if (strings.size() != 2) {
         return new MaterializedViewFreshness(true);
     }
 
+    /**
+     * @throws TableNotFoundException when underlying storage table of materialized view is not found
```

Comment:
where it's being thrown exactly? I looked at the code and couldn't find it exactly

Reply:
I didn't see any use of `TableNotFoundException` either, other than it is included in the method's Javadoc. Shall we remove it from the comment then?

Reply:
Ok. But can this method throw it?

Reply:
Removed since we could not find where `TableNotFoundException` is being thrown.